### PR TITLE
Add documentation on using existing security groups and creating new ones

### PR DESCRIPTION
**Pull request by Xenwar**
Add documentation on using existing security groups and creating new ones

### CODE REVIEW

File: capo/capo-security-groups.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  ]
+}
+```
+
+For pre-created security groups referenced in ```OpenStackMachineTemplate.spec.template.spec.securityGroups.name```, the details need to be retrieved from openstack itself.
+
+
+### Create new security groups
+
+As of now, there is no way of creating a security group using CAPO manifests. The security groups and their respective rules in use for the controlplane and worker servers are hard coded as show in [controlPlaneRules](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/8d40447beb68e9b199973ba42023a0ecdddc6f5c/pkg/cloud/services/networking/securitygroups.go#L149) and [workerRules](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/8d40447beb68e9b199973ba42023a0ecdddc6f5c/pkg/cloud/services/networking/securitygroups.go#L225)
```
Comment (furkatgofurov7) [inline capo/capo-security-groups.md:122]: typo: shown

**Review by furkatgofurov7 — COMMENTED** [review]
Small nit.
Otherwise LGTM.

**Review by fmuyassarov — COMMENTED** [review]
lgtm

File: capo/capo-security-groups.md (hunk 1 above)
Comment (Xenwar) [reply capo/capo-security-groups.md]: fixed

**Review by fmuyassarov — APPROVED** [review]

**Pull request merged**